assets/js/components/product-category-control/index.js

```diff
@@ -39,21 +40,54 @@ const ProductCategoryControl = ( {
 			? item.name
 			: `${ item.breadcrumbs.join( ', ' ) }, ${ item.name }`;
 
+		const listItemAriaLabel = showReviewCount
+			? sprintf(
+				_n(
+					'%s, has %d review',
+					'%s, has %d reviews',
+					item.reviews,
+					'woo-gutenberg-products-block'
+				),
+				accessibleName,
+				item.reviews
+			)
+			: sprintf(
+				_n(
+					'%s, has %d product',
+					'%s, has %d products',
+					item.count,
+					'woo-gutenberg-products-block'
+				),
+				accessibleName,
+				item.count
+			);
+
+			const listItemCountLabel = showReviewCount
```

Comment:
There is too much indentation here:
```suggestion
		const listItemCountLabel = showReviewCount
```